app/src/main.js

```diff
@@ -27,14 +27,22 @@ var vm = new Vue({
 });
 
 vm.$on("subregionSelected", function(d) {
+  if (d === "All") {
+    $(".map").css({ fill: "" });
+    this.$children[0].$children[4].$children[0].selectedRegion = {};
```

Comment:
I think there's a way we can avoid doing this children.children stuff. I'm working on figuring that out.